modules/logging-audit-log-filtering.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+//
+:_mod-docs-content-type: CONCEPT
+[id="logging_audit_filtering_{context}"]
+= Understanding audit filtering for log forwarding
+OpenShift API servers generate audit events for each API call, detailing the request, response, and the identity of the requester. While these events are crucial for security audits, they often contain more information than necessary, leading to unmanageably large volumes of data.
+
+The API Audit filter provides a solution to this problem by enabling the exclusion of non-essential events and the reduction of event size, facilitating a more manageable audit trail.
+
+An audit policy event contains meta-data describing who made the request. It can also include the full body of the API request, and the response that was sent. Rules are checked in order, checking stops at the first matching rule. The `level` of an audit rule determines how much data is included in the event:
+
+* None: The event is dropped.
+* Metadata: Only the audit metadata is included, request and response bodies are removed.
+* Request: Audit metadata and the request body are included, the response body is removed.
+* RequestResponse: All data is included: metadata, request body and response body. The response body can be very large. For example, `oc list -A pods` generates a response body containing the YAML description of every pod in the cluster.
```

Comment:
Seems `oc list` is not a valid command, I got:

```
$ oc list
error: unknown command "list" for "oc"

Did you mean this?
	get
	wait
```

Reply:
@QiaolingTang - fixed, thank you!